api_workflows/serializers.py

```
@@ -0,0 +1,416 @@
+            else:
+                workflow_data["export_format"] = None
+                check_result = check_engine.execute_check(
+                    db_name=workflow_data["db_name"], sql=sql_content
+                )
+        except serializers.ValidationError:
+            raise
+        except Exception:
+            logger.exception("Unexpected error while validating workflow submission.")
+            raise serializers.ValidationError(
+                {"errors": "An internal validation error occurred."}
+            )
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)